features/step_definitions/networking.rb

```diff
@@ -696,3 +685,35 @@
     raise "Fail to delete the ovs pod"
   end
 end
+
+Given /^the default interface on nodes is stored in the#{OPT_SYM} clipboard$/ do |cb_name|
+  ensure_admin_tagged
+  step "I select a random node's host"
+  node_name ||= node.name
+  cb_name = "interface" unless cb_name
+
+  sdn_pod = BushSlicer::Pod.get_labeled("app=sdn", project: project("openshift-sdn", switch: false), user: admin) { |pod, hash|
+    pod.node_name == node_name
+  }.first
+  cache_resources sdn_pod
+  @result = sdn_pod.exec("bash", "-c", "ip route show default | awk 'NR==1{print $5}'", as: admin)
```

Comment:
I think the code above can be replaced with:
```
step "I run command on the \"node_name\" node's sdn pod:", table(
  "| bash | -c | ip route show default | awk 'NR==1{print $5}' |"
)
```
delegating to the step avoids duplicating logic to find SDN pod.

Reply:
@anuragthehatter  I also added the comment in https://github.com/openshift/verification-tests/pull/502#discussion_r355254278 about this 
if we use sdn_pod to run this command `ip route`', then many cases using this step will cannot be run on OVN plugin.

Reply:
This is fixed with same step integerating OVN check